api/requirements.in

```diff
@@ -38,4 +38,5 @@ analytics-python
 django-lifecycle
 drf-writable-nested
 django-filter
-dataclasses
\ No newline at end of file
+dataclasses
+rudder-sdk-python==1.0.0b1
```

Comment:
Do we need to pin this to a beta release?

Reply:
Hmm yeah that beta is over a year old but doesnt look like theres much activity https://github.com/rudderlabs/rudder-sdk-python/releases - this was as per the install instructions on their side.